client/app/pages/users/show.html

```diff
@@ -59,6 +59,9 @@ <h3 class="p-l-5">{{user.name}}</h3>
           <input name="email" type="email" class="form-control" ng-model="user.email" required/>
           <error-messages input="userSettingsForm.email" form="userSettingsForm"></error-messages>
         </div>
+        <div class="form-group required">
+          <label class="control-label checkbox-inline"><input name="is_active" type="checkbox" ng-model="user.is_active" />Is Active</label>
+        </div>
```

Comment:
The button can be named "Disable" and make this a one way action for now (I doubt there will be many cases when a user gets disabled and then reinstated).